federation/apis/swagger-spec/extensions_v1beta1.json

```diff
@@ -94,7 +94,7 @@
      {
       "type": "v1beta1.Ingress",
       "method": "POST",
-      "summary": "create a Ingress",
+      "summary": "create an Ingress",
```

Comment:
swagger spec is autogenerated.
This change will be overwritten when someone runs hack/update-federation-swagger-spec.sh

The right way to fix it is to fix the code that generates it.
Filed https://github.com/kubernetes/kubernetes/issues/32305 with details.
Feel to reply on the issue if you want to take it up.